.env.example

```
@@ -42,6 +42,8 @@ MEMORIES_RETENTION_DAYS=180
 UPSTASH_REDIS_REST_URL=https://your-redis.upstash.io
 UPSTASH_REDIS_REST_TOKEN=your-upstash-token
 QSTASH_TOKEN=your-qstash-token
+# Idempotency: set to "false" to fail closed when Redis is unavailable (default is fail-open)
+IDEMPOTENCY_FAIL_OPEN=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDEMPOTENCY_FAIL_OPEN key should go before the QSTASH_TOKEN key

(UnorderedKey)